extensions-contrib/opentelemetry-emitter/src/main/java/org/apache/druid/emitter/opentelemetry/OpenTelemetryEmitterConfig.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.druid.emitter.opentelemetry;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import javax.annotation.Nullable;
+
+public class OpenTelemetryEmitterConfig
+{
+  @JsonProperty
+  private final String endpoint;
```

Comment:
I think, we can add here more options from here
But I'm not sure that we want
What do you think of it? @sirianni @xvrl @Zohimi

Reply:
The auto-configure that @IvanVan referenced seems pretty powerful. It might be quite expensive to replicate, especially since opentelemetry supports arbitrary plugins. It doesn't look like we can easily pass in the arbitrary params, though, since the DefaultConfigProperties constructor is private: https://github.com/open-telemetry/opentelemetry-java/blob/81963066b9635222ab5aecc20fd33b73b8d08f6f/sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/DefaultConfigProperties.java#L31-L52

@xvrl should we consider relying on the auto-configure via system properties/environment variables rather than trying to recreate all the config options as a jackson module?

Reply:
yes, I think we should only configure things specific to the extension here, and let someone define the auto-configuration configs like they would for any other app using auto-configuration.